sentences/sk/light_HassTurnOff.yaml

```diff
@@ -6,3 +6,4 @@ intents:
           - "(<turn_off> | <turn_off_light>) [(všetky | všetko)] [(svetlo | svetlá | osvetlenie | lampu | lampy)] [(v | vo | na | pred | pod | pri)] <area>"
         slots:
           domain: light
+        response: "lights_area"
```

Comment:
Ano, takto. Jen ještě při generování to hodí i větu "Zapni světlo v <area>, což je už jedna entita. Otázka je, zda se to pak nebude tlouct s větou v zapnutí jedné entity.

 - sentences:
          - "(<vypnout>|<zhasnout>) světlo <area>"
          - "<area> (<vypnout>|<zhasnout>) světlo"
        slots:
          domain: light
        response: "light"
      - sentences:
          - "(<vypnout>|<zhasnout>) [všechna] světla <area>"
          - "<area> (<vypnout>|<zhasnout>) [všechna] světla"
        slots:
          domain: light
        response: "lights_area"

Reply:
jj toto ma tiez napadlo ze by mohol byt problem

Reply:
problém nie ale byť sa to momentálne bude, to je myslím to o čom sa bavíme v diskusii. Navrhujem toto mergnut a rozdeliť to následne.

Reply:
Užívá se ve slovenštině nějaká věta, třeba `zapni všecko světlo v ...` pro označení světel? Je to gramaticky správně?
Pokud ne, pak není problém slovo `světlo` vyhodit a dát ho jako další slot pro označení jediného světla.

Reply:
rozhodně se dá vyhodit `lampu` a tu dát do svého slotu - ta nikdy použitá jako entita pro celou místnost nebude.

Reply:
rozdelil som to, takze toto sa moze mergnut a potom sa moze mergnut aj moj pull request 
https://github.com/home-assistant/intents/pull/764 

kvoli "nedostatocnym pravam" som to mohol spravit  len takto